src/components/EnvironmentVariables/index.js

```diff
@@ -0,0 +1,427 @@
+                    <th>Value</th>
+                  </Collapse>
+                </tr>
+              </thead>
+              <tbody>
+                {displayProjectVars.map((projEnvVar, index) => {
+                  return (
+                    <Fragment key={index}>
+                      <tr>
+                        <td className="varName">{projEnvVar.name}</td>
+                        <td className="varScope">{projEnvVar.scope}</td>
+                        {prjLoading ? (
+                          <Collapse in={openPrjVars}>
+                            <td className="varValue" id={index}>
+                              <div className="loader"></div>
+                            </td>
+                          </Collapse>
+                        ) : projEnvVar.value ? (
+                          <Collapse in={openPrjVars}>
+                            <td className="varValue" id={index}>
+                              {projEnvVar.value.length <= 100 &&
+                              !prjValueState[index] ? (
+                                <div className="showHideContainer">
+                                  {hashValue(projEnvVar.value).substring(0, 25)}
+                                  <a
+                                    href="#"
+                                    onClick={() => prjValuesShow(index)}
+                                  >
+                                    <Image
+                                      src={show}
+                                      className="showHide"
+                                      style={{ all: "unset" }}
+                                      alt=""
+                                    />
+                                  </a>
+                                </div>
+                              ) : projEnvVar.value.length <= 100 &&
+                                prjValueState[index] ? (
+                                <div className="showHideContainer">
+                                  {projEnvVar.value}
+                                  <a
+                                    href="#"
+                                    onClick={() => prjValuesHide(index)}
+                                  >
+                                    <Image
+                                      src={hide}
+                                      className="showHide"
+                                      style={{ all: "unset" }}
+                                      alt=""
+                                    />
+                                  </a>
+                                </div>
+                              ) : projEnvVar.value.length >= 100 &&
+                                !prjValueState[index] ? (
+                                <div className="showHideContainer">
+                                  {hashValue(projEnvVar.value).substring(0, 25)}
+                                  <a
+                                    href="#"
```

Comment:
same with using spans instead or you could actually add the click handler to the image